group_vars/all.yml

```diff
@@ -20,8 +20,8 @@ storyprotocol:
       ports:
         suffix:
           auth_rpc: 51
-          p2p: 03
-          network_discovery: 03
+          p2p: 30
```

Comment:
any reason to change this ? lets keep the suffix same as the original port unless its already in use